erlang/plan.sh

```diff
@@ -1,28 +1,29 @@
 pkg_name=erlang
 pkg_origin=core
-pkg_version=20.2
+pkg_version=21.3
 pkg_description="A programming language for massively scalable soft real-time systems."
 pkg_upstream_url="http://www.erlang.org/"
-pkg_dirname=otp_src_${pkg_version}
+pkg_dirname="otp_src_${pkg_version}"
 pkg_license=('Apache-2.0')
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
-pkg_source=http://www.erlang.org/download/otp_src_${pkg_version}.tar.gz
-pkg_filename=otp_src_${pkg_version}.tar.gz
-pkg_shasum=24d9895e84b800bf0145d6b3042c2f2087eb31780a4a45565206844b41eb8f23
+pkg_source="https://github.com/erlang/otp/archive/OTP-${pkg_version}.tar.gz"
```

Comment:
In general, I think we should try to keep the pkg_source in line with what upstream provides as their official download location which is `erlang.org/download`.